src/utils/date.ts

```diff
@@ -0,0 +1,24 @@
+import dayjs from 'dayjs';
+
+export const dateFormatMonth = 'MMMM YYYY';
+export const dateFormatDay = 'DD MMMM YYYY';
+
+type DateFormat = 'day' | 'month';
+
+const isCurrentMonth = (date: string) => {
+  const currentMonth = dayjs().format(dateFormatMonth);
+
+  return dayjs(date).format(dateFormatMonth) === currentMonth;
+};
+
+export const formatDate = (date: string, format: DateFormat) => {
+  return format === 'day' ? dayjs(date).format(dateFormatDay) : dayjs(date).format(dateFormatMonth);
+};
+
+export const displayCurrentIfCurrentDate = (date: string, format: DateFormat) => {
```

Comment:
Maybe it would be cleaner if you put the logic of this function into `formatDate`?

Reply:
I gave it a shot 😄